Fmaraev/Hometask_3/task_1.py

```diff
@@ -0,0 +1,18 @@
+"""Найти самое длинное слово в введенном предложении. Учтите что в предложении
+есть знаки препинания."""
+
+# Вводим предложение и удаляем все знаки препинания
+st = input('Enter a sentence ')
+x = ',.!?:;()[]{}#$%^&*+-/|><='
+for i in x:
+    st = st.replace(i, '')
```

Comment:
It wouldn't work correctly if we forgot space between i.e. bracket and `word(like this)` - word would be `wordlike`